gcovr/configuration.py

```diff
@@ -1027,6 +1027,11 @@ def merge_options_and_set_defaults(partial_namespaces, all_options=None):
         type=int,
         default=1,
     ),
+    GcovrConfigOption(
+        "exclude_pattern_prefix", ["--exclude-pattern-prefix"],
+        help="Define the regex prefix used in markers / line exclusions (i.e _EXCL_START, _EXCL_START, _EXCL_STOP)",
+        type=str
+    ),
```

Comment:
Set the default prefix in the the argument parser.
```suggestion
    GcovrConfigOption(
        "exclude_pattern_prefix", ["--exclude-pattern-prefix"],
        help="Define the regex prefix used in markers / line exclusions (i.e _EXCL_START, _EXCL_START, _EXCL_STOP)",
        type=str,
        default=r"[GL]COVR?",
    ),
```

Reply:
Pushed.